site/content/en/docs/Installation/_index.md

```diff
@@ -29,10 +22,15 @@ If no dedicated nodes are available, Agones will run on regular nodes.
     [recommended set of admission controllers](https://kubernetes.io/docs/admin/admission-controllers/#is-there-a-recommended-set-of-admission-controllers-to-use).
 - Firewall access for the range of ports that Game Servers can be connected to in the cluster.
 - Game Servers must have the [game server SDK]({{< ref "/docs/Guides/Client SDKs/_index.md"  >}}) integrated, to manage Game Server state, health checking, etc.
+
+{{< alert title="Warning" color="warning">}}
+Later versions of Kubernetes may work, but this project is tested against 1.12, and is therefore the supported version.
+Agones will update its support to n-1 version of what is available across all major cloud providers - GKE, EKS and AKS
+{{< /alert >}}
 {{% /feature %}}
 
 {{% feature publishVersion="1.2.0" %}}
-- Kubernetes cluster version 1.13
+- **Kubernetes cluster version 1.13**
     - [Minikube](https://github.com/kubernetes/minikube), [Kind](https://github.com/kubernetes-sigs/kind), [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/),
```

Comment:
This is the only place I can see on the website where we mention KIND. I think we should remove it from here (since there aren't any instructions to use it) and just leave it in the github documentation.

Reply:
Valid point. Also, I don't think anyone is testing on Kind currently.

Reply:
The kind link is still in here. Did you mean to remove it?